Readability.js

```diff
@@ -1703,6 +1703,17 @@ Readability.prototype = {
    */
   isProbablyReaderable: function(helperIsVisible) {
     var nodes = this._getAllNodesWithTag(this._doc, ["p", "pre"]);
+    if (this._doc.querySelectorAll && Array.from) {
+      nodes = Array.from(nodes);
+      var brs = this._doc.querySelectorAll("br:nth-of-type(5)");
+      for (var i = 0; i < brs.length; i++) {
```

Comment:
Should we limit this to more than just brs.length? What if we have a page that has a ton of brs?